### PR TITLE
Added a break out for MapLoom

### PR DESCRIPTION
**Pull request by theduckylittle**
Lets a docker based dev environment use a development version of MapLoom

### CODE REVIEW

**Comment by harts-boundless** [comment]
I think we should do whatever is required to make the frontend workflow more comfortable. I wish I had more time to help with this, but we can merge it as-is and fix things up later if you need it.

Can the work be done in the startup script instead of `Dockerfile`?

Can we use `pip install -e /mnt/django-exchange-maploom` instead of directly putting symlinks into the virtualenv? Then in the worst case if you have to do symlink magic, it is in `/mnt/django-exchange-maploom` instead of directly in the virtualenv, or I'm not sure, maybe the symlinks can even be managed in the host. 

Basically I am trying, under the constraint of not increasing the daily friction for frontend work, to reduce the rate at which all the magical host mount stuff does confusing things e.g. because of garbage files that arise in working directories, or because of forgetting what branch you had a share on ... I have already contemplated getting rid of all the mounts altogether because of the problems they have caused.

**Comment by harts-boundless** [comment]
Or maybe `django-exchange-maploom` could use a submodule or something? Just throwing out ideas

**Comment by theduckylittle** [comment]
***WARNING!*** MapLoom will need to be built after cloning! Otherwise you are likely to be shown the following error:

```
 File "/mnt/exchange/manage.py", line 30, in <module>
   execute_from_command_line(sys.argv)
 File "/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
   utility.execute()
 File "/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
   self.fetch_command(subcommand).run_from_argv(self.argv)
 File "/env/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
   self.execute(*args, **cmd_options)
 File "/env/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
   output = self.handle(*args, **options)
 File "/env/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 168, in handle
   collected = self.collect()
 File "/env/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 107, in collect
   handler(path, prefixed_path, storage)
 File "/env/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 305, in copy_file
   if not self.delete_file(path, prefixed_path, source_storage):
 File "/env/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 255, in delete_file
   self.storage.delete(prefixed_path)
 File "/env/lib/python2.7/site-packages/django/core/files/storage.py", line 288, in delete
   os.remove(name)
OSError: [Errno 21] Is a directory: '/scratch/static_root/maploom'
```

To build MapLoom (assumes npm and bower are installed):
```
$ cd ~/boundless/MapLoom
$ npm install ; bower install ; grunt build
````
